### PR TITLE
bug(applet): fix resource not found bug #617

### PR DESCRIPTION
**Pull request by hunshenshi**
#617

### CODE REVIEW

File: pkg/modules/applet/applet.go [hunk 1]
```diff
@@ -256,23 +270,9 @@ func Update(ctx context.Context, r *UpdateReq) (*UpdateRsp, error) {
 			if r.Info.WasmCache != nil {
 				rb = &deploy.CreateReq{Cache: r.Info.WasmCache}
 			}
-			ins, err = deploy.UpsertByCode(ctx, rb, raw, ins.State, ins.InstanceID)
+			ins, err = deploy.UpsertByCode(ctx, rb, raw, enums.INSTANCE_STATE__STARTED, ins.InstanceID)
```
Comment (Liuhaai) [inline pkg/modules/applet/applet.go:273]: shoud the state be changed to `START` in applet Update?

File: cmd/srv-applet-mgr/apis/middleware/current_user.go [hunk 2]
```diff
@@ -113,12 +113,16 @@ func (v *CurrentAccount) WithAppletContextBySFID(ctx context.Context, id types.S
 		ctx = types.WithInstance(ctx, ins)
 	}
 
-	res, err := resource.GetBySFID(ctx, app.ResourceID)
+	return ctx, nil
```
Comment (Liuhaai) [inline cmd/srv-applet-mgr/apis/middleware/current_user.go:116]: ctx order: prj -> app -> ins

File: pkg/modules/applet/applet.go (hunk 1 above)
Comment (hunshenshi) [reply pkg/modules/applet/applet.go:273]: 1. Setting the status to "start" is intended to confirm whether the wasm file can run properly.
2. Additionally, manually updating the applet is aimed at putting it into a running state.

File: cmd/srv-applet-mgr/apis/middleware/current_user.go (hunk 2 above)
Comment (hunshenshi) [reply cmd/srv-applet-mgr/apis/middleware/current_user.go:116]: use `appletID` to get the `applet`, then use `applet.ProjectID` to get `project`. 
So get the applet first.

**Pull request merged**